utils.py

```diff
@@ -58,8 +58,25 @@ def format_new_line_slow_zone(sz):
     return ret
 
 
+def format_percent_change(latest, previous):
+    ret = ""
+    percent_change = (latest - previous) / previous * 100.0
+    ret += "⬆️ " if percent_change < 0 else "⬇️ "
+    ret += str(abs(round(percent_change, 2))) + " %\n"
+    return ret
+
+
+def format_updated_slow_zone_details(sz):
+    ret = ""
+    ret += get_stop_pair(sz) + "\n"
+    ret += "⏳ " + str(round(sz["latest_delay"], 1)) + "s "
+    ret += "-> " + str(round(sz["previous_delay"], 1)) + "s "
+    ret += format_percent_change(sz["latest_delay"], sz["previous_delay"])
+    return ret
+
+
 def format_time(t):
-    return datetime.strptime(t, "%Y-%m-%dT%H:%M:%S")
+    return datetime.strptime(t, "%Y-%m-%dT%H:%M:%SZ")
```

Comment:
I seriously think we need to do something different here. Because this morning the lambda ran and the json doesn't have the `Z` again. We need to do something that works no matter what

Reply:
Yeah we can't just keeping going back and forth on this. Why does the `json` formatting change? Ideally it should be consistent and we shouldn't have to handle this on the bot end.

Reply:
It seems like what the lambda deploys and what gets deployed from a local run is different. I'm not quite sure why other than maybe a Linux vs macos thing?